### PR TITLE
docs: add example to vim.ui.select

### LINKED ISSUE #17137: Examples for vim.ui functions
### Feature already in Vim?

_No response_

### Feature description

It would be good if there are some examples for `vim.ui.select` and `vim.ui.input` . Something like this : 

```lua
lua vim.ui.select({"hi", "set buftype?", "set filetype?"}, {}, function (choice) vim.cmd(choice) end)
```

### PR DESCRIPTION
**Pull request by dundargoc**
Closes https://github.com/neovim/neovim/issues/17137

### CODE REVIEW

**Comment by dundargoc** [comment]
99% sure that the part about `Defaults to Input: .` is wrong by the way.

**Review by mfussenegger — APPROVED** [review]
> 99% sure that the part about Defaults to Input: . is wrong

yes I think you're right.

File: runtime/lua/vim/ui.lua [hunk 1]
```diff
@@ -57,6 +75,13 @@ end
 ---               Called once the user confirms or abort the input.
 ---               `input` is what the user typed.
 ---               `nil` if the user aborted the dialog.
+---
+--- Example:
+--- <pre>
+--- vim.ui.input({ prompt = 'Select value for shiftwidth: ' }, function(input)
```
Comment (bew) [inline runtime/lua/vim/ui.lua:81]: Hmm to me `Select ...` would mean to select among a set of choices, I think it would be more exact to say `Enter ...`

Comment (dundargoc) [reply runtime/lua/vim/ui.lua:81]: https://github.com/neovim/neovim/pull/17251

**Pull request merged**